lab-doug/app/cowsay.controller.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const angular = require('angular');
+const cowsay = require('cowsay-browser');
+/**
+ * app - description
+ *
+ * @param  {string} 'cowsayController' registering a controller name with the app
+ * @param  {string} ['$scope'        a list of dependencies for the controller
+ * @param  {variable} function($scope)  a list of parameters listed in the same order as the dependencies in the line above
+ * @return {value}                    returns the result of cowsay.say method  with "text" as parameter
+ */
+
+angular.module('cowsayApp').controller('cowsayController', ['$scope', function($scope){
```

Comment:
I would probably break this onto multiple lines, like
```javascript 
angular.module('cowsayApp')
  .controller('cowsayController', [
    '$scope', 
    CowsayController,
  ]);

  function CowsayController($scope){
    // stuff
  }

```
If you wanted to be super angular-y, you'd wrap the whole thing in an IIFE